p2p/src/net/default_backend/transport/impls/channel.rs

```diff
@@ -211,6 +211,7 @@ mod tests {
         net::default_backend::{transport::BufferedTranscoder, types::Message},
     };
 
+    #[tracing::instrument(skip(seed))]
     #[rstest::rstest]
     #[trace]
     #[case(Seed::from_entropy())]
```

Comment:
Would the `instrument` annotation cause the seed to be printed if the `skip(...)` was removed? Maybe we could get rid of the annoying `#[trace]` attributes before every `#[case]`

Reply:
In that case the seed will be included in every log line.
So no, it can't replace `#[trace]`.